board/freescale/mx6ullevk/mx6ullevk_siklu_ethernity.c

```diff
@@ -0,0 +1,41 @@
+/*
+ * mx6ullevk_siklu_ethernity.c
+ *
+ *  Created on: Jul 12, 2022
+ *      Author: yba@tkos.co.il
+ */
+
+#ifndef CONFIG_SPL_BUILD
+
+#include <common.h>
+#include <linux/ctype.h>
+#include <command.h>
+
+#include "siklu_def.h"
+#include "siklu_api.h"
+#include "cpld_reg.h"
+
+/*
+ *
+ */
+static int do_siklu_ethernity_reset(cmd_tbl_t * cmdtp, int flag, int argc,
+		char * const argv[]) {
+
+	int rc = 0;
+	T_CPLD_LOGIC_WD_RW_REGS reset_reg;
+
+	reset_reg.s.cfg_fpga_8020_rst = 1;
```

Comment:
What about all other bits in `uint8`. They are left uninitialized here. I guess the CPLD currently ignores these bits. But a future CPLD might behave strangely when random bits are set.

Reply:
Fixed. Thanks.